numpy/core/defchararray.py

```diff
@@ -1187,7 +1187,15 @@ def replace(a, old, new, count=None):
     See Also
     --------
     str.replace
+    
+    Examples
+    --------
+    >>> import numpy as np 
+    >>> print(np.char.replace('That is a mango', 'mango', 'banana'))
+    That is a banana
 
+    >>> print(np.char.replace('The dish is fresh', 'is', 'was'))
+    The dwash is fresh
```

Comment:
In case it helps, when I run this example locally I get:

`The dwash was fresh`

Reply:
yes that's correct, it must be 'The dwash was fresh'. I wanted to show the example for how 'is' in all places is replaced by 'was'. How can I change it?